contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -0,0 +1,860 @@
+        uint256 currentPoolWethShare,
+        uint256 requiredPoolWethShare,
+        uint256 WETHPositionBalance,
+        uint256 OETHbPositionBalance
+    ); // 0x6c6108fb
+
+    event PoolRebalanced(
+        uint256 currentPoolWethShare,
+        uint256 targetPoolWethShare,
+        uint256 WETHPositionBalance,
+        uint256 OETHbPositionBalance
+    );
+    
+    event PoolWethShareUpdated(
+        uint256 newWethShare
+    );
+
+    event PrincipalPositionBeforeSwap(
+        uint256 WETHPositionBalance,
+        uint256 OETHbPositionBalance
+    );
+
+    event PrincipalPositionAfterSwap(
+        uint256 WETHPositionBalance,
+        uint256 OETHbPositionBalance
+    );
+
+    event WithdrawLiqiudityShareUpdated(
+        uint128 newWithdrawLiquidityShare
+    );
+
+    event PoolWethShareVarianceAllowedUpdated(
+        uint256 poolWethShareVarianceAllowed
+    );
+
+    event LiquidityRemoved(
+        uint128 withdrawLiquidityShare,
+        uint256 removedWETHAmount,
+        uint256 removedOETHbAmount,
+        uint256 WETHAmountCollected,
+        uint256 OETHbAmountCollected,
+        uint256 underlyingAssets
+    );
+
+    event LiquidityTokenBurned(
+        uint256 tokenId
+    );
+
+    event LiquidityAdded(
+        uint256 WETHAmountDesired,
+        uint256 OETHbAmountDesired,
+        uint256 WETHAmountSupplied,
+        uint256 OETHbAmountSupplied,
+        uint256 tokenId,
+        uint256 underlyingAssets
+    );
+
+    /**
+     * @dev Verifies that the caller is the Governor, or Strategist.
+     */
+    modifier onlyGovernorOrStrategist() {
+        require(
+            msg.sender == governor() ||
+            msg.sender == IVault(vaultAddress).strategistAddr(),
+            "Not the Governor or Strategist"
+        );
+        _;
+    }
+
+    /// @notice the constructor
+    /// @param _stratConfig the basic strategy configuration
+    /// @param _wethAddress Address of the Erc20 WETH Token contract
+    /// @param _OETHbAddress Address of the Erc20 OETHb Token contract
+    /// @param _swapRouter Address of the Aerodrome Universal Swap Router
+    /// @param _nonfungiblePositionManager Address of position manager to add/remove
+    ///         the liquidity
+    /// @param _clPool Address of the Aerodrome concentrated liquidity pool
+    /// @param _sugarHelper Address of the Aerodrome Sugar helper contract
+    /// @param _lowerBoundingTick Smaller bounding tick of our liquidity position
+    /// @param _upperBoundingTick Larger bounding tick of our liquidity position
+    /// @param _tickClosestToParity Tick that is closer to 1:1 price parity
+    constructor(
+        BaseStrategyConfig memory _stratConfig,
+        address _wethAddress,
+        address _OETHbAddress,
+        address _swapRouter,
+        address _nonfungiblePositionManager,
+        address _clPool,
+        address _sugarHelper,
+        int24 _lowerBoundingTick,
+        int24 _upperBoundingTick,
+        int24 _tickClosestToParity
```

Comment:
This parameter isn't used in the constructor

Reply:
bad slip-up on my end thank you: https://github.com/OriginProtocol/origin-dollar/pull/2184/commits/376a376737c61fc7ce987f5ed5b355d149873d24